Source/Core/Core/PowerPC/BreakPoints.cpp

```diff
@@ -103,6 +150,7 @@ void BreakPoints::Add(u32 address, bool temp, bool break_on_hit, bool log_on_hit
   bp.break_on_hit = break_on_hit;
   bp.log_on_hit = log_on_hit;
   bp.address = address;
+  bp.condition = condition;
```

Comment:
```suggestion
  bp.condition = std::move(condition);
```

Reply:
I don't think that will do anything different for a const reference. That said, this function could accept a string by value.

Reply:
I clearly read your comments out of order.